tests/function/test_cli.py

```diff
@@ -134,7 +134,7 @@ def test_interval(self, virtwho):
             rhsm_log = virtwho.rhsm_log_get()
             assert "No data to send, waiting for next interval" in rhsm_log
         else:
-            assert result["loop"] == 60 or result["loop"] == 61
+            assert result["loop"] in [60, 61, 62, 63]
```

Comment:
`assert 60 <= result["loop"] <= 63` would be pythonic and better for the future's optimization

Reply:
done, thanks for the suggestion